tests/publiccloud/validate_repos.pm

```diff
@@ -38,6 +38,7 @@ sub run {
         my ($incident, $type);
         for my $maintrepo (@repos) {
             next unless validate_repo($maintrepo);
+            die "Development Tools Repository found: IBS Mirror does not clone development repos, so this can't be tested. Test internally.\n $maintrepo" if $maintrepo =~ /Development-Tools/;
```

Comment:
I would move this to the `validate_repo` subroutine.

Reply:
> I would move this into the `validate_repo` subroutine.

even if it is an IBS Mirror things only?

Reply:
Is this code only used by tests powered by IBS Mirror? Is it also used by tests directly using IBS?

Reply:
@mpagot I would not see why this being an IBS mirror thing would disqualify it from being move to `validate_repo` :thinking: 

My rationale is: It's part of the validation, therefore it should go into the validation subroutine.

Reply:
Pavel's point remains valid though.

Reply:
We decided to go another route, and not abort but instead record the sdk repo and skip it. This way `validate_repos` remains untouched, and the recording is moved to `cluster_add_repos`. 

i.e. https://openqa.suse.de/tests/12365737#step/cluster_add_repos/3

Reply:
Oh no, we scarred Bill away :scream: